lang/java/avro/src/test/java/org/apache/avro/io/TestEncoders.java

```diff
@@ -262,6 +262,7 @@ void arrayBackedByteBuffer() throws IOException {
   @Test
   void mappedByteBuffer() throws IOException {
     Path file = Paths.get(DIR.getPath() + "testMappedByteBuffer.avro");
+    file.toFile().deleteOnExit();
```

Comment:
```suggestion
    Path file = DIR.toPath().resolve( "testMappedByteBuffer.avro");
```
For example: here we have a `@TempDir` already present, and cleaned up automatically in the test case -- we just accidentally forgot the path separator so the file was _outside_!

Reply:
Ah, O.K. I'll try it.